libs/sr/db/src/Database/DataManagers/LoadoutDataManager.ts

```diff
@@ -24,6 +25,7 @@ export interface Loadout {
 
   buildTcIds: string[]
   optConfigId: string
+  bonusStats: Array<{ tag: Tag; value: number }>
```

Comment:
just curious, why are we using Array<> syntax over []?

Reply:
I think `{ tag: Tag; value: number }[]` looks jank, imo

Reply:
You can use `TagMapEntries<number>`